repo2docker/app.py

```diff
@@ -64,6 +64,7 @@ def _default_log_level(self):
 
     buildpacks = List(
         [
+            NixBuildPack,
```

Comment:
nix should probably be where the Julia and R build packs are in this list, not at the end here. The Dockerfile build pack is the ultimate escape hatch, so we need to give priority to it over all other build packs.